guides/doc-Upgrading_Project/docinfo.xml

```diff
@@ -1,7 +1,7 @@
 <title>Upgrading Red Hat Satellite</title>
 <productname>Red Hat Satellite</productname>
 <productnumber>6.14</productnumber>
-<subtitle>Upgrading Red Hat Satellite Server and Capsule Server</subtitle>
+<subtitle>Upgrade Satellite Server and Capsule</subtitle>
```

Comment:
```suggestion
<subtitle>Upgrade Satellite Server and Capsule to a new major release</subtitle>
```

Reply:
To keep it parallel to the Updating guide subtitle, if it makes sense.

Reply:
I wouldn't do this, because there should be "...to 6.14" in the main title. Also, an upgrade only happens once per major release, whereas updates and minor releases are multiple per major release.